just_audio/darwin/just_audio/Package.swift

```diff
@@ -0,0 +1,25 @@
+// swift-tools-version: 5.9
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+
+import PackageDescription
+
+let package = Package(
+    name: "just_audio",
+    platforms: [
+        .iOS("12.0"),
+        .macOS("10.14")
+    ],
+    products: [
+        .library(name: "just-audio", targets: ["just_audio"])
+    ],
+    dependencies: [],
+    targets: [
+        .target(
+            name: "just_audio",
+            dependencies: [],
+            cSettings: [
+                .headerSearchPath("include/just_audio")
```

Comment:
I believe you have missed out the code to define the AUDIO_SESSION_MICROPHONE env variable as you mentioned here: https://github.com/flutter/flutter/issues/161182#issuecomment-2586088292. 

I think this is causing the code which requires microphone access to be compiled which means that an entry in the Info.plist is being required when uploading a build to App Store Connect.